app/src/main/java/com/google/jetpackcamera/MainActivity.kt

```diff
@@ -89,4 +94,19 @@ private fun CameraPermission(cameraPermissionState: PermissionState) {
             Text("Request permission")
         }
     }
+}
+
+@Composable
+fun JetpackCameraNavHost(
```

Comment:
It would be better to separate the app construction from MainActivity to help with readability and adjustability of the code itself. Here is what I think:
* code for creating individual UI components unique to each feature should stay under a ui directory for that feature (i.e. ui for settings containers, list items, etc.)
*  constructed screens/ elements can be kept under the .../main/java/ui directory (i.e. settings screen, preview screen)
* a class with a method that that essentially constructs the app and connects the pages + data, contains this NavHost function. That function would be called here in MainActivity

Reply:
Incorporated the change through foundation branch